airflow/providers/dbt/cloud/hooks/dbt.py

```diff
@@ -244,26 +244,21 @@ async def get_job_details(
         endpoint = f"{account_id}/runs/{run_id}/"
         headers, tenant = await self.get_headers_tenants_from_connection()
         url, params = self.get_request_url_params(tenant, endpoint, include_related)
-        proxies = self._get_proxies(self.connection)
+        proxies = self._get_proxies(self.connection) or {}
+
         async with aiohttp.ClientSession(headers=headers) as session:
-            if proxies is not None:
-                if url.startswith("https"):
-                    proxy = proxies.get("https")
-                else:
-                    proxy = proxies.get("http")
-                async with session.get(url, params=params, proxy=proxy) as response:
-                    try:
-                        response.raise_for_status()
-                        return await response.json()
-                    except aiohttp.ClientResponseError as e:
-                        raise AirflowException(f"{e.status}:{e.message}")
-            else:
-                async with session.get(url, params=params) as response:
-                    try:
-                        response.raise_for_status()
-                        return await response.json()
-                    except aiohttp.ClientResponseError as e:
-                        raise AirflowException(f"{e.status}:{e.message}")
+            proxy = proxies.get("https") if proxies and url.startswith("https") else proxies.get("http")
+            extra_request_args = {}
+
+            if proxy:
+                extra_request_args["proxy"] = proxy
+
+            try:
+                async with session.get(url, params=params, **extra_request_args) as response:    # type: ignore[arg-type]
+                    response.raise_for_status()
+                    return await response.json()
+            except aiohttp.ClientResponseError as e:
+                raise AirflowException(f"{e.status}: {e.message}")
```

Comment:
why bother converting this to airflow exception?  i think best to avoid doing this unless there's a specific reason to change it.

Reply:
like, better IMO to just not have a try except here since we're raising in all cases anyway.  all it does is add a layer of translation / obfuscation

Reply:
True

Reply:
resolved in https://github.com/apache/airflow/pull/42840/commits/e75ce6c5d6e00ade29e4fd03f05bc19c45812980